src/domUpdates.js

```diff
@@ -158,15 +158,17 @@ const domUpdates = {
     show([modal, pantryView])
   }, 
 
-  exitModalView(modal, pantryView, shoppingCartView) {
-    hide([modal, pantryView, shoppingCartView])
-  }, 
+  exitModalView(modal, pantryView, shoppingCartView, errorMessage) {
+    hide([modal, pantryView, shoppingCartView, errorMessage])
+  },
 
   updateShoppingCartView(currentRecipe, shoppingCart, modal, shoppingCartView, user) {
     shoppingCart.innerHTML = '';
     currentRecipe.ingredients.forEach(ingredient => {
       let ing = user.pantry.ingredients.find(ing => ing.id === ingredient.id)
+      // let value;
```

Comment:
Possibly remove these commented out lines if we're not going to revisit this functionality.